exercises/robot-simulator/metadata.toml

```diff
@@ -0,0 +1,3 @@
+blurb = "Write a robot simulator."
+source = "Inspired by an interview question at a famous company."
+source_url = ""
```

Comment:
```suggestion
```

Slight preference - but somebody might argue that it shouldn't be done in this PR.

Reply:
Sorry I should have picked this before committing - misread email.

I think it's worth squeezing in - it's not a controversial change and we've already spent a lot of eyes-time on this issue.